commands/create-account.js

```diff
@@ -36,6 +36,39 @@ module.exports = {
     handler: exitOnError(createAccount)
 };
 
+const createAccountCommandDeprecated = {
+    command: 'create_account <accountId>',
+    builder: (yargs) => yargs
+        .option('accountId', {
+            desc: 'Unique identifier for the newly created account',
+            type: 'string',
+            required: true
+        })
+        .option('masterAccount', {
+            desc: 'Account used to create requested account.',
+            type: 'string',
+            required: true
+        })
+        .option('publicKey', {
+            desc: 'Public key to initialize the account with',
+            type: 'string',
+            required: false
+        })
+        .option('newLedgerKey', {
+            desc: 'HD key path to use with Ledger. Used to generate public key if not specified directly',
+            type: 'string',
+            default: "44'/397'/0'/0'/1'"
+        })
+        .option('initialBalance', {
+            desc: 'Number of tokens to transfer to newly created account',
+            type: 'string',
+            default: '100'
+        }),
+    handler: exitOnError(async (options) => {
+        console.log('near create_account is deprecated. Please use near create-account.');
```

Comment:
It might be nice if we say in which version of near-shell we plan to remove the old version

Reply:
Good idea! Let's give it 2 minor version bumps